packages/arb-token-bridge-ui/src/util/TokenListUtils.ts

```diff
@@ -179,15 +182,28 @@ export const validateTokenList = (tokenList: TokenList) => {
   return validate(tokenList)
 }
 
-export const addBridgeTokenListToBridge = (
-  bridgeTokenList: BridgeTokenList,
+export const addBridgeTokenListToBridge = ({
+  bridgeTokenList,
+  arbTokenBridge,
+  parentChainId,
+  childChainId
+}: {
+  bridgeTokenList: BridgeTokenList
   arbTokenBridge: ArbTokenBridge
-) => {
+  parentChainId: ChainId
+  childChainId: ChainId
+}) => {
   fetchTokenListFromURL(bridgeTokenList.url).then(
     ({ isValid, data: tokenList }) => {
       if (!isValid) return
-
-      arbTokenBridge.token.addTokensFromList(tokenList!, bridgeTokenList.id)
+      if (!tokenList) return
+
+      arbTokenBridge.token.addTokensFromList({
+        arbTokenList: tokenList,
+        listId: bridgeTokenList.id,
+        parentChainId,
+        childChainId
```

Comment:
passing chain id fixes the following issue:

user's source chain was an Arbitrum Chain, e.g. Arb One, and destination chain is L3 Orbit
then user switched destination to L1
the L1<->L2 Arb token was not added to `bridgeTokens` because the chain ids in `useArbTokenBridge` was still the old pair (Arb One & L3 Orbit) instead of L1 & L2